enterprise_subsidy/apps/transaction/api.py

```diff
@@ -86,15 +95,18 @@ def cancel_transaction_external_fulfillment(transaction):
             fulfillment_cancelation_successful = True
         else:
             logger.warning(
-                '[fulfillment cancelation] dont know how to cancel transaction %s with provider %s',
+                "[cancel_transaction_external_fulfillment] Don't know how to cancel transaction %s with provider %s",
```

Comment:
[nit] Don't know how --> Unable

Reply:
I'll defend the original wording: "Unable to" is accurate, but "don't know how" is the reason.  I think more explicit is good here.